Tiltfile

```diff
@@ -37,7 +37,11 @@ local_resource(
 local_resource(
   name="webchat",
   serve_dir="packages/inject",
-  serve_cmd="yarn serve",
+  cmd="mkdir -p dist && yarn copy:shareable",
```

Comment:
**note**: This fixes the webchat UI not being styled locally when serving from Tilt.